src/DaysAccordion.tsx

```diff
@@ -16,13 +16,21 @@ import ExercisesContext from './context/ExercisesContext';
 const DaysAccordion = () => {
   const { exercises } = useContext(ExercisesContext);
 
+  const mapFunction = ((exercise: Exercise) => (
```

Comment:
Need a more descriptive variable name. Given that the parameter is an exercise, and the return value is a table row, a good name would be `getExerciseTableRow`.